### PR TITLE
NEXUS-5023: Fixing mime types for WAR, EJB and EAR.

### PR DESCRIPTION
**Pull request by cstamas**
They were 'application/zip' and now they are 'application/java-archive'

### CODE REVIEW

**Comment by cstamas** [comment]
CI
https://builds.sonatype.org/view/nexus-features/job/nexus-oss-its-feature/369/

**Comment by peterlynch** [comment]
+1

**Comment by cstamas** [comment]
prev CI caught a UT that test exactly the "expectations" from change props file. UT fixed, new CI
https://builds.sonatype.org/view/nexus-features/job/nexus-oss-its-feature/370/

**Comment by cstamas** [comment]
linux passed, win failed with throttle control failure (is unrelated)

File: nexus/nexus-api/src/main/resources/mime-types.properties [hunk 1]
```diff
@@ -18,8 +18,8 @@ zip = application/zip
 jnlp = application/x-java-jnlp-file
 gz = application/x-gzip
 bz2 = application/x-bzip
-war = application/zip
-ejb = application/zip
-ear = application/zip
+war = application/java-archive
+ejb = application/java-archive
+ear = application/java-archive
```
Comment (jdillon) [inline nexus/nexus-api/src/main/resources/mime-types.properties:23]: probably should also include rar

**Comment by jdillon** [comment]
+1 (though should also include rar http://docs.oracle.com/javaee/1.4/tutorial/doc/Overview5.html )

**Pull request merged**